cmd/cassandra-reader/main.go

```diff
@@ -103,13 +118,27 @@ func connectToCassandra(cluster, keyspace string, logger logger.Logger) *gocql.S
 	return session
 }
 
-func connectToThings(url string, logger logger.Logger) *grpc.ClientConn {
-	conn, err := grpc.Dial(url, grpc.WithInsecure())
+func connectToThings(cfg config, logger logger.Logger) *grpc.ClientConn {
+	var opts []grpc.DialOption
+	if cfg.clientTLS {
+		if cfg.caCerts != "" {
+			tpc, err := credentials.NewClientTLSFromFile(cfg.caCerts, "")
```

Comment:
What's `tpc`? If I did not get this on a first sight, would that mean that naming is badly chosen?

Reply:
We have this name in every service, and it's short for `TransportCredentials`. Also, this is local var that is used in only one place, so this name is not so important (it is in go spirit to use short names for local vars).